.github/workflows/ci.yml

```
@@ -178,11 +178,29 @@ jobs:
         include:
           - os: ubuntu-24.04
             test-args: --features full,unstable --workspace
-          # Only test client on windows & mac (since its the only binaries supported for those os for now)
           - os: macos-14
-            test-args: --package mithril-client --package mithril-client-cli --features full,unstable
+            test-args: >
+              --features full,unstable --workspace
+              --exclude mithril-aggregator
+              --exclude mithril-signer
+              --exclude mithril-relay
+              --exclude mithril-client-wasm
+              --exclude mithril-aggregator-fake
           - os: windows-latest
```

A comment explaining why the list is larger would be nice
```suggestion
          # Windows have a larger list of exclusion, reason include that some tests are too slow (ie: mithril-stm) or can't compile (ie: mithril-common)
          - os: windows-latest
```

---

Done